setup.py

```diff
@@ -23,6 +23,8 @@
     install_requires=[
         "attrs>=21.3.0",
         "itemadapter>=0.2.0",
+        "typing-extensions>=4.3.0;python_version<'3.10'",
+        "web-poet @ git+https://git@github.com/scrapinghub/web-poet@facd6d0#egg=web-poet",
```

Comment:
To unblock merging of this PR: what do you think about adding web-poet as a dependency here (e.g. web-poet >= 0.4.0), but specifying the commit in tox.ini only? It seems this particular version is only needed to for tests, and a previous release would work fine, from the user point of view.

Reply:
Yep, that should be fine. 👍 https://github.com/zytedata/zyte-common-items/pull/11/commits/17f47b5328fb714911d641963c97943ff571db6d

I think if the user has installed `0.4.0` but then encounters a **mypy** error like `Skipping analyzing "web_poet": module is installed, but missing library stubs or py.typed marker`, then the `web-poet` version could simply be bumped in their project.